doc/actor.md

```diff
@@ -7,570 +8,521 @@ http://stixproject.github.io/data-model/1.2/ta/ThreatActorType/
 
 * This entry is required
 
-### Keyword Key
+* Keyword Key
+  * Plumatic Schema: :id
 
-* Plumatic Schema: :id
-
-### String Value
-
-The URI of this entity.
-
-* Plumatic Schema: Str
+* String Value
+  * The URI of this entity.
+  * Plumatic Schema: Str
 
 ## MapEntry: :type -> String
```

Comment:
I think the ":"'s make it confusing though.  It triggers some sort of tag matching on github, too.  I was thinking about ruby's hashmap literals, which in retrospect is "=>" and not "->".  What do you think about using "=>"?